src/main/java/ua/edu/internship/interview/web/handler/ErrorResponse.java

```diff
@@ -0,0 +1,25 @@
+package ua.edu.internship.interview.web.handler;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+import java.time.LocalDateTime;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.time.LocalDateTime'
<!-- __reviewdog__:Cg42Y2VjMmNhMjEyNGUzNRIKY2hlY2tzdHlsZQ== -->